app/controllers/products_controller.rb

```diff
@@ -10,7 +10,6 @@ def new
 
```

Comment:
Please use a method to get products, you are repeating this line  ` @product = Product.find(params[:id])`

Reply:
Ok